### DISCUSSION TITLE
delete policy

### DISCUSSION THREAD

**Discussion by igorg-il** [jobrunr/jobrunr#733] [Q&A] [answered]
hello
We are using jobrunr 6.1.2
I saw according to the jobrunr site that there is deleteOnFailure argument as Job annotation parameter
I don't see such argument when trying to add it in our code (there are only names, retries, labels, jobFilters(. Is it only available in pro version?
Also is there a way to make all failed jobs to be deleted similarly as done with succeeded ones (instead of doing it per job)?
thank you

**Comment by rdehuyss** [comment, marked as answer]
Hi @igorg-il , this is indeed only part of the Pro offering.
